mentorship ios/Extensions/UserDefaults.swift

```diff
@@ -1,9 +1,7 @@
 //
-//  UserDefaults.swift
-//  mentorship ios
-//
-//  Created by Yugantar Jain on 08/06/20.
-//  Copyright © 2020 Yugantar Jain. All rights reserved.
+// UserDefaults.swift
+// Created on 08/06/20.
+// Created for AnitaB.org Mentorship-iOS
```

Comment:
The filename("UserDefaults.swift") is also there in the header. We have omitted that in other files. 
I think we can keep it in every file.